cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -390,3 +398,25 @@ func (p *pod) KillTask(ctx context.Context, tid, eid string, signal uint32, all
 	})
 	return eg.Wait()
 }
+
+func (p *pod) DeleteTask(ctx context.Context, tid string) error {
+	t, err := p.GetTask(tid)
+	if err != nil {
+		return errors.Wrap(err, "could not delete task")
```

Comment:
While we will ultimately fail to delete the task if we can't perform this operation, ideally anybody who calls DeleteTask should annotate the error returned here with something generic in this light (e.g. "failed to delete %s task"). This should just state what failed internally (we failed to get the `tid` task).

Reply:
`GetTask` returns a `"task with id: '%s' not found"` error, and I didnt want to be repetitive with the error messages. Ideally the `GetExec` below would also not have a bland error, but I made it different from here for some reason.